crates/bevy_math/src/geometry.rs

```diff
@@ -25,17 +55,55 @@ impl<T: Default + Reflect + PartialEq> Default for Size<T> {
     }
 }
 
-/// A rect, as defined by its "side" locations
+/// A rectangle defined by its side locations.
+///
+/// It is commonly used to define positions, margins, paddings and borders.
```

Comment:
I think we should explain that the values supplied define the distances from the nominal "center" of the rect to the edge.

If any of the distances are negative, the center will not be contained within the rect.

Reply:
Implemented this and also added a description of the example to clarify how the `Rect` works.